packages/core/src/db/entity/postgresEntityStore.ts

```diff
@@ -191,36 +165,23 @@ export class PostgresEntityStore implements EntityStore {
     const fragments = [];
 
     if (where) {
-      const whereFragments: string[] = [];
-
-      for (const [field, value] of Object.entries(where)) {
+      const whereFragments = Object.entries(where).map(([field, value]) => {
         const [fieldName, rawFilterType] = field.split(/_(.*)/s);
-
-        // This is a hack to handle the = operator, which the regex above doesn't handle
+        // This is a hack to handle the "" operator, which the regex above doesn't handle
         const filterType = rawFilterType === undefined ? "" : rawFilterType;
-
-        const sqlOperators = sqlOperatorsForFilterType[filterType];
-        if (!sqlOperators) {
+        const sqlSymbols = sqlSymbolsForFilterType[filterType];
+        if (!sqlSymbols) {
           throw new Error(
             `SQL operators not found for filter type: ${filterType}`
           );
         }
 
-        const { operator, patternPrefix, patternSuffix, isList } = sqlOperators;
+        const whereValue = getWhereValue(value, sqlSymbols);
 
-        let finalValue = value;
+        return `"${fieldName}" ${whereValue}`;
+      });
 
-        if (patternPrefix) finalValue = patternPrefix + finalValue;
-        if (patternSuffix) finalValue = finalValue + patternSuffix;
-
-        if (isList) {
-          finalValue = `(${(finalValue as (string | number)[]).join(",")})`;
-        } else {
-          finalValue = `'${finalValue}'`;
-        }
-
-        whereFragments.push(`"${fieldName}" ${operator} ${finalValue}`);
-      }
+      console.log({ whereFragments });
```

Comment:
ooc did you mean to leave this here?

Reply:
no lol nice catch :D need a lint rule for this in CI